index.bs

```diff
@@ -188,7 +188,7 @@ Example implementations of Barcode/QR code detection are e.g. <a href="https://d
 [Exposed=(Window,Worker),
  Constructor(optional BarcodeDetectorOptions barcodeDetectorOptions)]
 interface BarcodeDetector {
-  readonly attribute FrozenArray<BarcodeFormat> supportedFormats;
+  static Promise<FrozenArray<BarcodeFormat>> getSupportedFormats();
```

Comment:
This should be a `Promise<sequence<BarcodeFormat>>`. `FrozenArray` is specifically for attributes.

Reply:
Doh, of course, done.